src/pages/Home/Home.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Home from './Home';
+
+describe('Home', () => {
+  it('renders home page', () => {
```

Comment:
(imho) In the read world this is a useless test.
It's too atomic. Requires resources to write and mantain it but don't test anything valuable.
Consider use the snapshot tests.

Reply:
removed not needed tests for now